lib/util.js

```diff
@@ -40,3 +42,16 @@ export const fileToDataURL = blob =>
     reader.onload = e => res(e.target.result)
     reader.readAsDataURL(blob)
   })
+
+export const formatCode = code =>
+  new Promise((resolve, reject) => {
+    try {
+      const formatted = prettier.format(code, {
```

Comment:
If `prettier.format` is synchronous please make `formatCode` just return the code, not a promise.

Reply:
How should we handle format errors if it's sync? return Error object in the catch block and delegate error handling in the Editor?